entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Can we actually remove the use of an entrypoint here, and start both of these services within the image build?

I would like to be able to build pytest containers with this as a base, but don't want to have to try and replicate the entrypoint in my containers.

Reply:
I don't fully understand what you want. The entrypoint is required here otherwise selenium server won't start. You can easily override the entrypoint in your image.

Reply:
@mshriver ^

Reply:
@quarckster I think its appropriate to address this in another PR, instead of here.

Ultimately I'd like to not have to 'override' the entrypoint and start nginx or the selenium server (via vncserver) in my 'downstream' image.  I'll investigate this and open a PR - starting these as services on 'boot', something to that effect.

Reply:
@mshriver afaik there are no services in containers, ENTRYPOINT is used to run applications on "boot". Ideally, it should be only one executable per container.